.github/CODEOWNERS

```diff
@@ -43,17 +43,14 @@
 # PRLabel: %Batch
 /sdk/batch/                                         @gingi @paterasMSFT @dpwatrous
 
-# PRLabel: %Communication
```

Comment:
Please leave the PRLabel comment (L46) and the path (L47) and just remove the associated people.

Reply:
Done